marshmallow_oneofschema/one_of_schema.py

```diff
@@ -146,11 +148,12 @@ def load(self, data, many=None, partial=None):
             exc = ValidationError(errors, data=data, valid_data=result)
             raise exc
 
-    def _load(self, data, partial=None):
+    def _load(self, data, partial=None, unknown=None):
```

Comment:
why not capture the args / kwargs and just spread through?

Reply:
I'm not really fan of using args and kwargs, because it makes code less readable (even when extending other package's code). You never know what to expect, what you have available etc. passing everything to args/kwargs (in most of the cases) is a bad habit tbh